FRW/Modules/Wallet/Empty/EmptyWalletView.swift

```diff
@@ -81,14 +81,14 @@ struct EmptyWalletView: View {
 
     var recentListContent: some View {
         VStack(alignment: .leading, spacing: 0) {
-            Text("registerd_accounts".localized)
+          Text("registerd_accounts".localized + ": (\(vm.profiles.count))")
```

Comment:
Let's put the count first, it should be ` 7 Registered Accounts` Instead of `Registered Accounts: 7`